### PR TITLE
Import disableContentUpdate

### PR DESCRIPTION
**Pull request by jcharaoui**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | https://github.com/wallabag/doc/issues/4
| Translation   | no
| Fixed tickets | 
| License       | MIT

Adding a disableContentUpdate option, which defaults to false, to allow users to control whether Wallabag will fetch content from the Internet during an import operation. Also fixes a few minor things.

Still WIP, we might want to add the option to the UI and API. Also the recent changes to ContentProxy require a good revamp of tests.

### CODE REVIEW

**Comment by jcharaoui** [comment]
A quick assessement would be appreciated. I'd like to continue working on tests, but I'd like to know if you're interested in merging this feature.

File: src/Wallabag/CoreBundle/Helper/ContentProxy.php [hunk 1]
```diff
@@ -158,15 +183,17 @@ public function assignTagsToEntry(Entry $entry, $tags, array $entitiesReady = []
...
+     * Validate that the given content has at least a title and a url.
      *
      * @param array $content
-     *
-     * @return bool true if valid otherwise false
      */
     private function validateContent(array $content)
     {
-        return isset($content['title']) && isset($content['html']) && isset($content['url']) && isset($content['language']) && isset($content['content_type']);
+        if (!(isset($content['title']) && !empty($content['title']))) {
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Helper/ContentProxy.php]: You can just use `!empty($content['title'])` (which handle isset too)

File: src/Wallabag/ImportBundle/Import/AbstractImport.php [hunk 2]
```diff
@@ -82,21 +83,43 @@ public function getMarkAsRead()
     }
 
     /**
+     * Set whether articles should be fetched for updated content.
+     *
+     * @param bool $markAsRead
```
Comment (j0k3r) [inline src/Wallabag/ImportBundle/Import/AbstractImport.php]: Wrong copy/paste

**Comment by j0k3r** [comment]
I'll wait for https://github.com/wallabag/wallabag/pull/3106 to be merged because it'll generate some conflict and some refacto I guess.
But for the moment, tests should be fine.

@jcharaoui will work on that PR again? :slightly_smiling_face:

**Comment by jcharaoui** [comment]
I hope to pick it up again, but not in the near future. If anyone wants to pick up where I left off, go right ahead!

**Comment by j0k3r** [comment]
Ok thanks for the quick feedback, I'll try to finish it then.

**Comment by j0k3r** [comment]
The only chance here is adding a new option to the import command: `disableContentUpdate` which disable fetching content from an url when enabled.
This should speed up any imports (because it'll save the entry without fechting any urls) but entry without content will still have no content.

Should we document that somewhere? (we don't have any documentation about commends except one which only list all commands)

**Comment by nicosomb** [comment]
We can start a more detailed documentation with this command.

**Comment by j0k3r** [comment]
Thanks @jcharaoui for starting the job!
It'll be available in 2.3

**Pull request merged**